Tryouts/Prototypes/Shell/Manifest/ManifestParser.cs

```diff
@@ -20,26 +20,28 @@
 using System.Text.Json;
 using System.Windows.Automation;
 using System.Windows.Controls.Primitives;
+using System.Windows.Documents;
 
 namespace Manifest
 {
-    internal class ManifestParser
+    internal static class  ManifestParser
     {
-        internal ManifestModel Manifest { get; set; }
+        internal static ManifestModel Manifest { get; set; }
```

Comment:
Unnecessary property. Why would you have a static parser that also stores the last manifest it has parsed?